openmetadata-ui/src/main/resources/ui/src/components/Modals/EntityDeleteModal/EntityDeleteModal.tsx

```diff
@@ -92,12 +90,15 @@ const EntityDeleteModal = ({
       }
       width={600}>
       <div data-testid="body-text">
-        <Typography className="mb-2">
-          {bodyText ||
-            t('message.delete-entity-permanently', {
-              entityType: getTitleCase(entityType),
-            })}
-        </Typography>
+        <div className="mb-2">
+          <Transi18next
+            i18nKey="message.permanently-delete-metadata"
+            renderElement={<strong data-testid="username" />}
```

Comment:
```suggestion
            renderElement={<strong data-testid="entityName" />}
```

Also strong will add `font-weight: 700` we are not using it in our application can you make it 500?

Reply:
yes addressed